### ISSUE TITLE
Unable to run a container which is reachable on ports 80 or 443

### ISSUE THREAD

**Issue by jhollmannk** [rancher-sandbox/rancher-desktop#1194]
### Rancher Desktop Version

0.7.1

### Rancher Desktop K8s Version

1.22.5

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 Enterprise 21H2

### What CPU architecture are you using?

x64

### Actual Behavior

When I start Rancher-Desktop for the first time after the start of my PC, I cannot use the standard ports 80 or 443 as a mapped port for a container.

The first time start seems to be important because the problem vanished after I switched from dockerd to containerd and back (and exited and startet Rancher-Desktop again). I wrote a detailed description in the additional information part.

### Steps to Reproduce

Start Rancher-Desktop with the dockerd container runtime enabled. Run the command
`docker run -it --rm -p 80:80 nginx`

### Result

The container is not started and the error response is:
`docker: Error response from daemon: driver failed programming external connectivity on endpoint priceless_agnesi (f96c48b8190b4874131ae68ba2bde839958f91b1e4c0eb905da7851033cbd383): Error starting userland proxy: listen tcp4 0.0.0.0:6443: bind: address already in use.`

### Expected Behavior

The container should run and be accessible via port 80 (or 443).

### Additional Information

I snooped around a bit and tcpview showed a wslhost.exe process that listens on port 80.
That lead to a process rancher-desktop-guestagent in the rancher-desktop WSL VM that had both ports (80 and 443) open.

As we had some dificulties reaching a started nginx with containerd/nerdctl, I switched to containrd to see If the same problem was there too. but it wasn't. So I switched back to dockerd and lo and behold: the rancher-desktop-guestagent was started again and was running but did not use ports 80 and 443. Now the described docker command ran without any issues.

Then I completely exited and started Rancher-Desktop again and still no ports 80 and 443 used by the rancher-desktop-guestagent process.

I know this is not the best reproducible item, but maybe you can look into the rancher-desktop-guestagent process to see why it would open the standard ports and whether that is really necessary (seemingly they are not really used).

**Comment by rfay** [comment]
This happens on macOS as well, it's a problem. I'm using dockerd.

**Comment by rfay** [comment]
From [Daniel Schaaff in slack](https://rancher-users.slack.com/archives/C0200L1N1MM/p1642109725136500?thread_ts=1642102962.132900&cid=C0200L1N1MM)

> It is due to the traefik ingress controller k3s includes. You can create a lima override file (not sure of the location on windows) with the contents

```
env:
  K3S_EXEC: --disable=traefik
```
> Then run 
```
helm helm uninstall traefik -n kube-system
$ helm uninstall traefik-crd -n kube-system
```

**Comment by jhollmannk** [comment]
Now when I think about it, it makes total sense that the standard ports are not readily available in the standrad configuration of RancherDesktop.

In the meantime I made some test deployments in the kubernetes part of RancherDesktop which were then reachable on the default http port, so I guess I was too focused on the docker use case when I created this issue.

Haven't tried to uninstall the traefik ingress controller because in the end I want to use both parts (kubernetes and docker with docker-compose) and will see to it that the services running with docker-compose will not use the standard ports (I can live with that).

**Comment by dougbreaux** [comment]
D'oh. I want to be able to use RD without it taking those ports. Although, based on what you're saying, perhaps #985 would address as well?

But also, why _sometimes_ RD grabs the ports and sometimes not, seems a problem any way you look at it?

**Comment by rfay** [comment]
Yeah, I think this is a significant problem and should remain open.

**Comment by jandubois** [comment]
Re-opening for further analysis

**Comment by swquinn** [comment]
I just ran into this as well and I was not expecting it (definitely caused a head scratch moment for a while until I had an "Ah ha!" moment). Is the Lima override currently the suggested work around, or is there a better approach until a more permanent solution is figured out?

**Comment by chevdor** [comment]
At least, :80 could serve, instead of a 404, some kind of explanation about what is "suddently" (after RD install) serving :80. As mentioned by others, my first unsuccesful test was a simple nginx test, on :80 obviously... and this simple test failed until I realized that :80 was now taken by RD although the main entry page shows nothing interesting by default.

**Comment by A-Shevchenko** [comment]
Also wondering how to do that with dockerd runtime?
Stopping Kubernetes as suggested in [FAQ](https://docs.rancherdesktop.io/faq/) actually breaks DNS resolution in containers run by `docker run`

**Comment by gunamata** [comment]
Option to enable/disable traefik is provided in release 1.1.0. With this feature, one can disable traefik from GUI and free up port 80 for alternate ingress configuration. With traefik disabled, you can also get around the original issue posted in the thread where the first time Rancher Desktop launch after a machine restart doesn't allow to use port 80.

Please give the latest Rancher Desktop version 1.1.1 a try and let us know if you still see this issue.

**Comment by jhollmannk** [comment]
Yes, this works now!
(Tried in a Windows installation)

**Comment by gunamata** [comment]
Closing this as the enable/disable traefik feature introduced in 1.1.0 solves the issue.

**Comment by foobarbecue** [comment]
Is there a way to expose a web server on the host with kubernetes disabled? Not working for me, just wondering if it's supposed to work.
